src/lib/Room/RoomFooter/RoomFooter.vue

```diff
@@ -471,12 +481,15 @@ export default {
 				})
 			}
 		},
-		onChangeInput() {
+		onChangeInput(preventTypingEvent=false) {
```

Comment:
```suggestion
		onChangeInput(shouldFireTypingEvent=true) {
```

É melhor colocar uma lógica "positiva" aqui e negar no teste, caso contrário pode ficar bem difícil de ler o código.